docs/migrating-from-travis-to-circle-ci.md

```diff
@@ -0,0 +1,53 @@
+# CircleCI
```

Comment:
Reading this doc, it sounds mostly like a summary of the CircleCI docs so I'm not sure if there's as much value in publishing it as I'd expected. We may leave this out from master.

Reply:
what exactly do you want to see in the docs? Do you want the commands to come through?

Reply:
I tried to break down the gist of the circleci docs as everything is very well explained there, has good examples. I added necessary links to the docs which are relevant to what we have used. Let me know what else do we need there :)